bot/cogs/commands/useful.py

```diff
@@ -636,9 +636,29 @@ async def image_search(self, ctx: Ctx, *, query):
         await ctx.reply_embed(ctx.l.useful.search.nope)
         return
 
-    @commands.command(name="remindme", aliases=["remind"])
+    @commands.group(name="reminder", aliases=["remind"])
     @commands.cooldown(1, 2, commands.BucketType.user)
-    async def remind_me(self, ctx: Ctx, duration_str: str, *, reminder: str):
+    async def reminder_parent(self, ctx: Ctx):
+        if not ctx.invoked_subcommand:
+            user_reminders = await self.db.fetch_user_reminders(ctx.author.id)
+            embed = discord.Embed(color=self.bot.embed_color)
+            embed.set_author(
+                name=f"{ctx.author.display_name}'s reminders", icon_url=ctx.author.avatar.url
+            )
+
+            for reminder in user_reminders:
+                unix_timestamp = format_dt(reminder["at"], style="R")
+                reminder["reminder"] = shorten_text(reminder["reminder"], 75)
+                embed.add_field(
+                    name=f"`#{reminder['id']}` - {unix_timestamp}",
+                    value=reminder["reminder"],
+                    inline=False,
+                )
+            await ctx.send(embed=embed)
```

Comment:
It would be nice if we invert this if statement so we can dedent the code within

```
if ctx.invoked_subcomman:
    return
    
...
```